spec/router_spec.js

```diff
@@ -142,6 +143,68 @@ describe("Router", function() {
     });
   });
 
+  describe("navigate", function() {
+    class MyRouter extends Router {
+    }
+
+    class MyView extends View {
+      render() {
+        this.$el.html('');
+        return this;
+      }
+    }
+
+    beforeEach(function() {
+      this.router = new MyRouter({ $el: $('<div>') });
+    });
+
+    it("should call router.navigate right away if view does not define a beforeNavigate function", function() {
```

Comment:
Using "should" is a bit of fluff. The same meaning could be communicated using: `it("calls ...`.